internal/common/runtime/runtime.go

```diff
@@ -17,6 +17,8 @@
 package runtime
```

Comment:
This is also stutter. Consider that if there's a package that only contains a go file named identically with the package, you may not need the package.